src/NovelRT/CMakeLists.txt

```diff
@@ -202,9 +199,11 @@ if(WIN32)
       Winmm
   )
 
-  if(NOVELRT_INCLUDE_CONAN_PATHS)
-    add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
+  # I hate CMake I hate it I hate it I hate it
+  # (work around for CMAKE REFUSING TO COPY NECESSARY FILES FOR DEBUGGING MAKING IT IMPOSSIBLE TO DEBUG)
+  add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
       ${CMAKE_BINARY_DIR}/thirdparty
       $<TARGET_FILE_DIR:Engine>)
-  endif()
+  file(GLOB libs RELATIVE ${CMAKE_BINARY_DIR}/thirdparty ${CMAKE_BINARY_DIR}/thirdparty/*.dll)
```

Comment:
So normally this would be bad, but because CMake hates everybody who touches it, I gave up and just did the easy thing here. We're getting the contents of the `thirdparty` folder setup by Conan, and setting that as a list of resource files. That's what the whole copy_build_products system uses to copy additional "non-primary" outputs.

Reply:
This will break Vulkan. Please include all config extensions as well.

Reply:
So at the moment, this does not break on my machine with Vulkan SDK installed based on this branch. Do you mean for the future? We currently tell conan to copy `*.json` files, but it doesn't report that it copies any during configuration AFAIK (and there appears to be no dependency on them based on this branch or main).

Reply:
Did some digging and spoke to @RubyNova - previously we used a `vulkan-validationlayers` conan package that produced JSON files at install, but we no longer use that package in mainline.

Reply:
OK, then disregard what I said previously. This is resolved.